neopixel.py

```diff
@@ -199,6 +199,30 @@ def set_pixel(self, pixel_num, rgb_w, how_bright=None):
         else:
             self.pixels[pixel_num] = pix_value
 
+    def get_pixelRGB(self, pixel_num):
+        """
+        Get red, green and blue value of pixel on position <pixel_num>
+
+        :param pixel_num: Index of pixel to be set or slice object representing multiple leds
```

Comment:
This does not currently work for slices. It can stay as is, but then this should says only "Index of pixel" without "or slice object representing multiple leds"

Reply:
Removed mention of slices as suggesed.